### ISSUE TITLE
Sample State

### ISSUE THREAD

**Issue by pvulakh** [jessicaf1/pinthefail#3]
Good job on this! A few changes to make:

### misc
- all keys should be `camelCase` :)

### `users`
- What does `pinnedBoards` mean, the boards that the user is at the creator of?
- [ ] remove the `pinnedPins` array since a user can't pin pins without pinning them to a board. The array of board ids in the users slice will give access to the user's boards, each of which will have an array of `pinIds` 
- [ ] add `followedUserIds` and `followedBoardIds` array

### `boards`
- [ ] add `description`

### `pins`
- [ ] remove `boardId`
- [ ] add `link_url` for the pins that take you somewhere outside the site when clicked

**Comment by jessicaf1** [comment]
yep, pinnedBoards meant the user-created boards. is that necessary/correct here?

**Comment by pvulakh** [comment]
Awesome, let's rename this to `boardIds` just so it's shorter. We definitely need this in the user slice, you're spot on in including this!
